homeassistant/components/renault/renault_hub.py

```diff
@@ -49,17 +50,32 @@ async def async_initialise(self, config_entry: ConfigEntry) -> None:
         self._account = await self._client.get_api_account(account_id)
         vehicles = await self._account.get_vehicles()
         if vehicles.vehicleLinks:
-            for vehicle_link in vehicles.vehicleLinks:
-                if vehicle_link.vin and vehicle_link.vehicleDetails:
-                    # Generate vehicle proxy
-                    vehicle = RenaultVehicleProxy(
-                        hass=self._hass,
-                        vehicle=await self._account.get_api_vehicle(vehicle_link.vin),
-                        details=vehicle_link.vehicleDetails,
-                        scan_interval=scan_interval,
+            await asyncio.gather(
```

Comment:
Note that some web services don't like concurrent requests. So make sure to test that it works well.

Reply:
Tests have shown that the scan interval is important - but concurrent requests are not.